module/tools/SystemConfiguration/data/system/default/etc/motd

```diff
@@ -0,0 +1,26 @@
+[38;5;15m[48;5;15m████████[48;5;0m▌[38;5;0m█[38;5;15m▟[48;5;15m████████[48;5;0m▙[48;5;15m██████████████████[38;5;0m[48;5;0m████████[38;5;214m▜[38;5;15m[48;5;215m▜[48;5;15m█[48;5;222m▙[38;5;214m[48;5;237m▖[38;5;15m[48;5;238m▝[48;5;15m▜█████[48;5;223m▛[38;5;214m[48;5;178m▛[38;5;235m[48;5;234m▘[38;5;15m[48;5;235m▐[48;5;15m█████████[38;5;247m[48;5;236m▘[38;5;234m[48;5;234m████[38;5;172m[48;5;235m▟[38;5;214m[48;5;0m▘[38;5;0m█[0m
+[38;5;15m[48;5;15m██[48;5;0m▌[38;5;0m█[38;5;15m▗▟[48;5;15m██[48;5;0m▌▐[48;5;15m███[38;5;0m[48;5;0m████[38;5;15m[48;5;15m███[48;5;0m▌[38;5;0m██[38;5;15m[48;5;15m███[38;5;0m[48;5;0m██[38;5;15m▐[48;5;15m███[48;5;0m▄▄[38;5;0m█████████████[38;5;214m▐[38;5;15m[48;5;215m▝[48;5;15m██[48;5;214m▖[38;5;214m[48;5;235m▖[38;5;15m[48;5;236m▝[48;5;145m▀[48;5;7m▀[48;5;8m▀[38;5;255m[48;5;235m▀[38;5;130m[48;5;234m▀[38;5;235m▘[38;5;234m█[38;5;8m▝[38;5;15m[48;5;239m▜[48;5;15m██████▛[48;5;236m▘[38;5;234m[48;5;234m███[38;5;94m▗[38;5;214m[48;5;136m▟[38;5;0m[48;5;0m███[0m
+[38;5;15m[48;5;15m████████[48;5;0m▖▐[48;5;15m██[48;5;0m▌[38;5;0m████[38;5;15m▐[48;5;15m██[48;5;0m▛[38;5;0m██[38;5;15m[48;5;15m███[38;5;0m[48;5;0m███[38;5;15m▝[48;5;15m████████[38;5;0m[48;5;0m███████████[38;5;214m▜[38;5;15m[48;5;215m▝[48;5;230m▌[38;5;214m[48;5;214m█[38;5;172m[48;5;237m▙[38;5;234m[48;5;234m██████████[38;5;239m▝[38;5;255m[48;5;235m▀[38;5;254m▀[38;5;239m▘[38;5;251m▄[38;5;255m[48;5;243m▟[38;5;223m[48;5;214m▘[38;5;214m██▛[48;5;0m▘[38;5;0m████[0m
+[38;5;15m[48;5;15m██[48;5;0m▌[38;5;0m██[38;5;15m▗[48;5;15m███[48;5;0m▝[48;5;15m███[48;5;0m▖[38;5;0m██[38;5;15m▗[48;5;15m███[48;5;0m▌[38;5;0m██[38;5;15m[48;5;15m███[38;5;0m[48;5;0m███[38;5;15m▄▙[38;5;0m████[38;5;15m[48;5;15m███[48;5;0m▌[38;5;0m████████████[38;5;214m▝[48;5;214m▜█[38;5;136m[48;5;234m▌[38;5;234m█████████[38;5;235m▗[38;5;15m[48;5;179m▝[38;5;223m[48;5;214m▀[38;5;215m▚[38;5;230m▄[38;5;15m[48;5;222m▞[48;5;214m▘[38;5;214m[48;5;0m▛[38;5;0m███████[0m
+[38;5;15m[48;5;15m████████[48;5;0m▌[38;5;0m█[38;5;15m▝[48;5;15m████████[48;5;0m▘[38;5;0m███[38;5;15m[48;5;15m███[38;5;0m[48;5;0m██[38;5;15m▜[48;5;15m████████[48;5;0m▛[38;5;0m█████████████████[38;5;214m▀▜[48;5;94m▙[48;5;236m▄[38;5;172m[48;5;234m▄[38;5;136m▄[38;5;130m▄▄[38;5;172m▄[38;5;214m[48;5;237m▄[38;5;223m[48;5;214m▀[38;5;215m▘[38;5;214m[48;5;0m▀▘[38;5;0m██████████[0m
+ _   _  _    _  _             _        
```

Comment:
Is there a way we can do a MOTD script instead of this? so it can generate based on the hostname?

Reply:
either port the ampscii stuff to C++ or make it easy to call the python scripts

Reply:
I'm fine just `cat`ing this file and then doing some banner thing for the hostname. Just so it's identical for all robots (since hostname is where it comes from)

Reply:
You happy with that change?